Code/Engine/RendererFoundation/Descriptors/Descriptors.h

```diff
@@ -239,10 +239,10 @@ struct ezGALTextureResourceViewCreationDescription : public ezHashableStruct<ezG
 struct ezGALBufferResourceViewCreationDescription : public ezHashableStruct<ezGALBufferResourceViewCreationDescription>
 {
   ezGALBufferHandle m_hBuffer;
-  ezUInt32 m_uiFirstElement = 0;
-  ezUInt32 m_uiNumElements = 0;
-  ezEnum<ezGALResourceFormat> m_Format = ezGALResourceFormat::Invalid;
-  bool m_bRawView = false;
+  ezUInt32 m_uiByteOffset = 0;                    ///< Start of the view to the buffer. Must be multiple of the element size.
+  ezUInt32 m_uiByteCount = 0;                     ///< If 0, this maps to the rest of the buffer.
```

Comment:
Vulkan has this VK_WHOLE_SIZE which maps to uint(-1) aka ezInvalidIndex, I think this would make more sense than 0 as magic value.

Reply:
Added `EZ_GAL_WHOLE_SIZE`.